src/gpuarray_blas_opencl_clblas.c

```diff
@@ -56,6 +56,10 @@ static void teardown(gpucontext *ctx) {
     clblasTeardown();
 }
 
+static const char *error(gpucontext *ctx) {
+  return "(clblas) error in blas call, no details for now.";
```

Comment:
This shouldn't be called when there are no errors.  I don't want to be too vague at this point and this is just to have something there.

I can spend another half-hour to get something about equivalent to the cublas thing.